cmd/access/main.go

```diff
@@ -24,6 +25,8 @@ func main() {
 		anb.Logger.Fatal().Err(err).Send()
 	}
 
+	nodeBuilder.PreInit(cmd.DynamicStartPreInit)
```

Comment:
It's a bit confusing to have `PreInit` _after_ `Initialize`, but this seems like correct usage.

Reply:
Changed it up a bit https://github.com/onflow/flow-go/pull/1893/commits/a24cc2a5241a68045645e4d1203580e1f759b6ae